docs/guide/blog/comment-blog.md

```diff
@@ -0,0 +1,619 @@
+* Querying for blog posts that have associated comments
+* Deleting comments from a blog post
+* Implementing logic for writing comments to the blockchain
+
+**Note:** For this tutorial, adding comments is available only to blog posts that are no older than 100 blocks. The 100 block value has been hard coded for rapid testing. You can increase the block count to a larger number to achieve a longer time period before commenting is disabled.
+
+## Prerequisites
+
+- This tutorial is an extension of and requires completion of the [Module Basics: Build a Blog](index.md) tutorial. 
+
+## Core Concepts 
+
+This tutorial relies on the `blog` blockchain that you built in the build a blog tutorial. 
+
+## Fetch Functions Using List Command
+
+To get the useful functions for this tutorial, you use the `starport scaffold list comment --no-message` command. Make sure to familiarize yourself with the command.
+
+1. Navigate to the `blog` directory that you created in the build a blog tutorial.
+
+2. To create the source code files to add CRUD (create, read, update, and delete) functionality for data stored as an array, run:
+
+```bash
+starport scaffold list comment --no-message
+```
+
+The `--no-message` flag disables CRUD interaction messages scaffolding because you will write your own messages.
+
+The command output shows the files that were created and modified:
+
+```text
+create proto/blog/comment.proto
+modify proto/blog/genesis.proto
+modify proto/blog/query.proto
+modify vue/src/views/Types.vue
+modify x/blog/client/cli/query.go
+create x/blog/client/cli/query_comment.go
+create x/blog/client/cli/query_comment_test.go
+modify x/blog/genesis.go
+modify x/blog/genesis_test.go
+create x/blog/keeper/comment.go
+create x/blog/keeper/comment_test.go
+create x/blog/keeper/grpc_query_comment.go
+create x/blog/keeper/grpc_query_comment_test.go
+modify x/blog/module.go
+modify x/blog/types/genesis.go
+modify x/blog/types/genesis_test.go
+modify x/blog/types/keys.go
+
+🎉 comment added.
+```
+
+
+## Add a Comment to a Post
+
+To create a new message that adds a comment to the existing post, run:
+
+```bash
+starport scaffold message create-comment postID:uint title body
+```
+
+The `starport scaffold message` command accepts `postID` and a list of fields as arguments. The fields are `title` and `body`.
+
+Here, `postID` is the reference to previously created blog post.
+
+The `message` command has created and modified several files:
+
+```text
+modify proto/blog/tx.proto
+modify x/blog/client/cli/tx.go
+create x/blog/client/cli/tx_create_comment.go
+modify x/blog/handler.go
+create x/blog/keeper/msg_server_create_comment.go
+modify x/blog/types/codec.go
+create x/blog/types/message_create_comment.go
+create x/blog/types/message_create_comment_test.go
+
+🎉 Created a message `create-comment`.
+```
+
+As always, start your development with a proto file. 
+
+In the `proto/blog/tx.proto` file, edit `MsgCreateComment` to:
+
+- Add `createdAt`
```

Comment:
We're saying "Add `createdAt`", but there is no `createdAt` in the code block below.

Reply:
Also, if we're adding a field to a newly scaffolded message, shouldn't we just have `createdAt` in the scaffolding command?